### ISSUE TITLE
Gauge with callable label_value using deprecated function

### ISSUE THREAD

**Issue by quaterneon** [rycus86/prometheus_flask_exporter#21]
This is running in python3.7 + flask, on a mac.

Abbreviated (and anonymized) setup is:
```
@app.route('URL_RULE', methods=['POST'])
@metrics.gauge('name',
               'description',
               labels={'group_id': lambda: request.view_args.get('group_id')},
               multiprocess_mode='livesum')
def bulk_post(group_id: str):
    pass
```

I get the following warning:
```
src/api/app.py:642: in <module>
    multiprocess_mode='livesum')
../../../../.local/share/virtualenvs/venv/lib/python3.7/site-packages/prometheus_flask_exporter/__init__.py:369: in gauge
    before=lambda metric: metric.inc()
../../../../.local/share/virtualenvs/venv/lib/python3.7/site-packages/prometheus_flask_exporter/__init__.py:426: in _track
    ) if labels else tuple()
../../../../.local/share/virtualenvs/venv/lib/python3.7/site-packages/prometheus_flask_exporter/__init__.py:425: in <genexpr>
    for key, call in labels.items()
../../../../.local/share/virtualenvs/venv/lib/python3.7/site-packages/prometheus_flask_exporter/__init__.py:418: in label_value
    if inspect.getargspec(f).args:
/usr/local/Cellar/python/3.7.2_2/Frameworks/Python.framework/Versions/3.7/lib/python3.7/inspect.py:1078: in getargspec
    DeprecationWarning, stacklevel=2)
E   DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
```

**Comment by rycus86** [comment]
Thanks a lot for reporting @quaterneon !
I'll have a look and try to switch to the non-deprecated way.

**Comment by rycus86** [comment]
This should be fixed in version `0.7.2` - will be available in a few minutes. Thanks again!
